test/test-package-tree.ts

```diff
@@ -0,0 +1,137 @@
+import test from 'ava';
+import {
+    getPackageTreeFromDependencyList, generatePackageTree
+} from '../src/package-tree';
+test(
+    'getPackageTreeFromDependencyList should return a populated packageTree array',
+    async t => {
+        const dependencies = {
+            'module1': 'version1',
+            'module2': 'version2'
+        };
+        const packageLock = {
+            "dependencies": {
+                'module1': {
+                    'requires': {
+                        'module5': 'version5'
+                    }
+                },
+                'module2': {
+                    'requires': {
+                        'module6': 'version6'
+                    }
+                },
+                'module3': {
+                    'requires': {
+                        'module4': 'version4'
+                    }
+                },
+                'module5': {},
+                'module6': {}
+            }
+        };
+        const result = await getPackageTreeFromDependencyList(dependencies, packageLock);
+        const expectedResult = [{
+                'rootPackageName': 'module1',
+                'version': 'version1',
+                'data': undefined,
+                'dependencies': [{
+                    'rootPackageName': 'module5',
+                    'version': 'version5',
+                    'data': undefined,
+                    'dependencies': []
+                }]
+            },
+
+            {
+                'rootPackageName': 'module2',
+                'version': 'version2',
+                'data': undefined,
+                'dependencies': [{
+                    'rootPackageName': 'module6',
+                    'version': 'version6',
+                    'data': undefined,
+                    'dependencies': []
+                }]
+            }
+        ];
+        t.deepEqual(result, expectedResult);
+    });
+
+    test(
+        'getPackageTreeFromDependencyList should return a populated packageTree array',
```

Comment:
The test message should uniquely identify the behavior being tested. This looks like the same string as in the previous test. How can you re-phrase this to show that the individual behavior being tested at a glance?